server.c

```diff
@@ -1300,6 +1457,250 @@ static int get_my_addr_str(int sk)
 	return 0;
 }
 
+#ifdef _WIN32
```

Comment:
All of this should be in windows specific files. Generally, any use of ifdef in the C code, if it can be avoided, should be avoided.

Reply:
The big chunk of code noted here is now moved.  It didn't reduce the number of ifdef significantly but they are smaller chunks.  If you have any ideas on how to avoid the remaining ifdefs, I can look into it.